README.md

```diff
@@ -113,25 +113,35 @@ Build instructions
 
 - Install the latest CMake from https://cmake.org/download/
 - Download Win64 Apache 2.4 from https://www.apachelounge.com/download/ (or use your own version)
-- Extract it to this folder (so there is a folder called Apache24 inside this folder)
+- note build options should match Apache build (architecture, variant, and runtime linking)
 
-**Open command prompt:**
+**cmake options:**
 
-`mkdir build`
+- USE_STATIC_RUNTIME (default is OFF) and will use /MD and /MDd. If ON will build with /MT and /MTd for release and debug, respectively.

```

Comment:
Default to ON, but recommend using it with OFF with the requirement of either
- Apache being compiled with the same Visual Studio version
- VC redist is installed

Reply:
I understand that this is a breaking change but using /MD as the default is consistent with the builds at apache lounge. In addition it is consistent with the default behavior of visual studio and cmake. VS redistributable is backward compatible now 2019 to 2017.